ballerina/tests/07_document_validation.bal

```diff
@@ -44,10 +44,10 @@ function dataProviderDocumentValidation1() returns (string[][]) {
     dataProvider: dataProviderDocumentValidation2
 }
 isolated function testDocumentValidation2(string documentFileName) returns error? {
-    string document = check getGraphQLDocumentFromFile(string `${documentFileName}.graphql`);
+    string document = check getGraphQLDocumentFromFile(appendGraphqlExtension(documentFileName));
     parser:Parser parser = new (document);
     _ = check parser.parse();
-    json expectedPayload = check getJsonContentFromFile(string `${documentFileName}.json`);
+    json expectedPayload = check getJsonContentFromFile(appendJsonExtension(documentFileName));
```

Comment:
can't we do `getJsonContentFromFile(documentFileName)`?  the `appendJsonExtension` can be called inside the `getJsonContentFromFile` function

Reply:
oh! this was a mistake. :grimacing: I'll update the PR.

Reply:
at the first sight, I thought it was a mistake. But there are still some test cases that are not updated to use the data provider(Ex: subscription test). Those test cases use the function with the complete file name. (Ex: `getJsonContentFromFile("inline_fragment.graphql")`.

Reply:
I think here the extension should be `.json`.  In this case we can update the lines by removing the extension getJsonContentFromFile("inline_fragment") as the function name clearly mention what kind of file we are reading

Reply:
yes. already did the change.